BGL/examples/BGL_LCC/normals.cmd

```diff
@@ -0,0 +1 @@
+cube.off
```

Comment:
seems to me that all the cmd files added should end with _lcc other the test script won't use them.